src/pages/api/games/index.ts

```diff
@@ -1,17 +1,82 @@
 import { customErrorHandler } from "@/utils/exceptions";
-import { createGame, getAllGames } from "../../../server/db/actions/GameAction";
+import {
+  createGame,
+  getSelectedGames,
+} from "../../../server/db/actions/GameAction";
 import { gameSchema } from "../../../utils/types";
 import { NextApiRequest, NextApiResponse } from "next";
+import { z, RefinementCtx } from "zod";
+//Ensure its an integer
+// GameContentEnum
+const GameContentEnum = {
+  answerKey: "answerKey",
+  parentingGuide: "parentingGuide",
+  lessonPlan: "lesson",
+  videoTrailer: "videoTrailer",
+};
```

Comment:
Declare this as an actual enum for type safety: see TS [docs](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)